File: digitalWareAngular/digitalWare/src/app/models/cliente.ts
import { claseBase } from "./claseBase";
import { factura } from "./factura";

export class cliente extends claseBase {
    nombre: string;
    apellido: string;
    edad: number;
    cedula: string;
    fechaUltimaCompra: Date | string;
    /**Dias promedio entre compras */
    frecuenciaCompra: number;
    facturas: factura[];
}

File: DigitalWare.Models/Cliente.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DigitalWare.Models
{
    public class Cliente: ClaseBase
    {
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public int Edad { get; set; }
        public string Cedula { get; set; }
        public DateTime? FechaUltimaCompra { get; set; } = null;
        /// <summary>
        /// Dias promedio entre compras
        /// </summary>
        public int FrecuenciaCompra { get; set; }
        public List<Factura> Facturas { get; set; }
    }
}
